client/my-sites/plan-features-2023-grid/components/header-price.tsx

```diff
@@ -112,7 +112,7 @@ const HeaderPriceContainer = styled.div`
 
 	.plan-price.is-large-currency {
 		.plan-price__integer {
-			font-size: 30px;
+			font-size: 28px;
```

Comment:
**Note:**

IDR header prices will overflow in the plans comparison grid if calculated with add-on prices. To address this, we decrease font-size slightly